app/client-app/src/app/pages/Wordlist/components/hooks/useWordlistSortingAndFiltering.js

```diff
@@ -1,7 +1,35 @@
 import { useSortingAndFiltering } from "hooks/useSortingAndFiltering";
 
-export const useWordlistSortingAndFiltering = (data) =>
-  useSortingAndFiltering(data, "word", {
+export const useWordlistSortingAndFiltering = (data) => {
+  const filterers = {
+    wordContains: "word",
+    minLengthIsMet: (value, filter) => value.word.length >= filter,
```

Comment:
I think you can merge minLength and maxLength into a single filter that checks the value is between both values